### DISCUSSION TITLE
LuiExtended Development Discussion

### DISCUSSION THREAD

**Discussion by ArtOfShred** [DakJaniels/LuiExtended#231] [General]
## 👋 Hail steel warriors!
I figure this might be a half decent option for communicating a bit vs. just commenting on ESOUI & pull requests. 
* Anyone is welcome to chime in here!

**Comment by ArtOfShred** [comment]
@saenic Hey I wanted to ask - I noticed in 6.4.0 you removed the Buff/Debuff overrides from the default game Effects Menu & most of the SpellCastBuffs tooltip functions. Was there anything wrong with them (UI errors/etc) or was it just because a lot of the tooltips were wrong now since I hadn't updated them for several years to match the ability changes?

**Reply by saenic** [reply]
Just because they were overriding the default tooltips. At this time there were lots of changes to abilities.
One reason you gave up was the high amount of maintenance each patch and this was an effort to reduce it.

Edit: iirc there were also some small bugs with them. And I personally prefered the default tooltips.

**Reply by ArtOfShred** [reply]
Okay I figured as much, the tooltip stuff definitely took a mountain of time. 

There was some code in the Effects Menu override that also hid a lot of the wierd dummy buffs/debuffs or other erroneous things that LUIE hides on the buff bars. I think I did something to hide those wrapped around -duration effects that bugged out and showed up too. Maybe worth considering restoring that functionality at some point.

**Comment by sharath-kd** [comment]
Hi! I've been using LUI Extended 7.2.6.3 and ran into something while trying
to track ESO status effects.

LUI's Prominent Debuffs works really well for me for several effects. I've
confirmed that these are currently working:

- Burning
- Poisoned
- Hemorrhaging
- Off Balance

However, I can't get some of the other ESO status effects to show up in the
Prominent Debuff tracker:

- Chilled — ID 95136
- Concussed — ID 95134
- Sundered — ID 178123
- Overcharged — ID 178118
- Diseased — ID 178127

Hemorrhaging is also listed as ID 148801, and it already works for me, so I'm
including it as a working example rather than asking for it to be added.

The missing effects themselves are definitely being applied. For example,
I can see Chilled through the normal ESO UI when it occurs, but it doesn't
appear as an effect that I can add/use in LUI's Prominent Debuffs.

While searching online, I found the ESOUI discussion ["Issue Tracking All Status Effects"](https://www.esoui.com/forums/showthread.php?t=11031), which discusses the same problem. From
what I gather, some of these status effects aren't exposed through the normal
EVENT_EFFECT_CHANGED event and instead need to be handled through
EVENT_COMBAT_EVENT.

I also noticed that EVENT_COMBAT_EVENT has a UnitId, which creates another
complication when trying to match the event to the player's current reticle
target.

Would it be possible to add broader support for these combat-event status
effects to LUI's existing Target Debuff / Prominent Debuff system?

It would be great if they could use the existing LUI display layout just like the other debuffs.

I completely understand if the UnitId/reticle-target issue makes this
difficult or requires a different approach. I mainly wanted to share what
I've found in case this is something that could be supported in a future
version.

Thanks for maintaining LUI Extended!

**Reply by DakJaniels** [reply]
Thank you for bringing this up, those status effects will be trackable in the update slated for Update 51(Sept-28-2026).

**Reply by sharath-kd** [reply]
Appreciate the response. Thanks for all the hard work.
